packages/react/src/auto/shadcn/GadgetShadcnTailwindSafelist.ts

```diff
@@ -98,7 +114,6 @@ export const GadgetShadcnTailwindSafelist = [
   "whitespace-nowrap",
   "z-0",
   "z-10",
-  "z-100",
   "z-20",
   "z-30",
   "z-50",
```

Comment:
not for right now, but someday it'd be nice to have tailwind generate this for us, or a test that ensures its accurate, I imagine folks are gonna forget to update it when they use other classes